### ISSUE TITLE
Dividend Address in user interface

### ISSUE THREAD

**Issue by ghost** [Peershares/Peershares#5]
In the Receive coins and Address Book tabs in client, create a third column titled Dividend Address. Display the Peercoin address in the new column. Do this in a way that doesn't destroy the code's ability to display just the original two columns (Label and Address). The reason for this is that in a later phase a dropdown will be added that allows the wallet to display either Peercoins or one of list of Peershare types. This depends on Task 3 being complete.
